CONTRIBUTING.md

```diff
@@ -383,6 +383,37 @@ tested it becomes an officially supported Akka feature.
 
 [List of Akka features marked as may change](http://doc.akka.io/docs/akka/current/common/may-change.html)
 
+## Java APIs
+
+Each feature in Akka must have an API for convenient usage from both Scala and Java, as we develop the Java APIs in
+a subset of Scala that is easy to use from Java there are ... 
+
+When new APIs are added or existing ones are significantly changed such changes should have test coverage written in 
+actual Java code.
```

Comment:
As an extra precaution I think we should create a standalone test project with all our dependencies (snapshot) in a pom.xml and create a script to copy most of such java sources from akka to that project. Then open it in Eclipse and IntelliJ to verify that there are no errors. IDE tools can be more picky than javac. This is difficult to completely automate, but a sanity check before releasing would be good enough.